packages/ckeditor5-paste-from-office/docs/features/paste-from-word.md

```diff
@@ -73,10 +72,18 @@ By default, CKEditor 5 will support pasting rich-text content from these applica
 
 You can find more information regarding compatibility with other applications in [this ticket](https://github.com/ckeditor/ckeditor5/issues/1184#issuecomment-409828069).
 
-If you think that support for any of the applications needs improvements, please add 👍 and comments in the following issues:
+If you think that support for any of the applications needs improvements, please add 👍&nbsp; and comments in the following issues:
 
 * [Support pasting from Excel](https://github.com/ckeditor/ckeditor5/issues/2513).
 * [Support pasting from Libre Office](https://github.com/ckeditor/ckeditor5/issues/2520).
 * [Support pasting from Pages](https://github.com/ckeditor/ckeditor5/issues/2527).
 
 Feel free to open a [new feature request](https://github.com/ckeditor/ckeditor5/issues/new/choose) for other similar applications, too!
+
+## Known issues
+
+Sometimes if the document pasted contains both images and styled text (e.g. headers), it may happen that the images are not pasted properly. This was noticed on certain macOS configurations and should not cause problems for Windows, however.
+
+If the image is included into the content with the `<v:shape><v:imagedata src="...."/></v:shape>` syntax it will not be pasted either, as this notation is not yet supported by CKEditor 5.
+
+We are currently working on handling this error. Refer to [this GitHub issue](https://github.com/ckeditor/ckeditor5/issues/2493#issuecomment-749507634) for more details.
```

Comment:
Why are you promising anything? We are not handling this - we have just analyzed why it does not work, and we chose to document this because there does not seem to be much that we can do at the moment. In the docs we can advise the users to try to paste just the image (as documented by the reporter of the issue and also mentioned by Kuba):

> OTOH if the copied document fragment does include an image but there are no styled elements (like heading 1) the `text/rtf` is available so an image is pasted properly.

Reply:
> We are not handling this

An attempt to display a placeholder if the image was not pasted (linked in the is handling of sorts.  
  
Of course I'll rewrite it.

Reply:
Information removed in commit.